main.go

```diff
@@ -400,3 +412,30 @@ func runScorecardSettings(githubEventName, scorecardPolicyFile, scorecardResults
 	}
 	return &result, nil
 }
+
+func signScorecardResult(scorecardResultsFile string) error {
+	err := os.Setenv("COSIGN_EXPERIMENTAL", "true")
```

Comment:
nit: you could inline this with the error check
`if err := ... ; err != nil { }`